portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,35 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+body, html {
+  height: 100%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.bg {
+    background-image: url("/images/mountain.JPG");
+    filter: blur(4px);
+    -webkit-filter: blur(4px);
+    height: 100%;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
 }
+
+#home_content {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-family: 'Garamond';
```

Comment:
Nit: Consistently use double-quotes.

Reply:
I think the style guide actually calls for single quotes in CSS (and double quotes in HTML).

Reply:
https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks

Reply:
Add single quotes in CSS

Reply:
add single quotes

Reply:
change to single quotes